clipmenu

```diff
@@ -22,9 +21,25 @@ Environment variables:
 - $CM_HISTLENGTH: specify the number of lines to show in dmenu/rofi (default: 8)
 - $CM_LAUNCHER: specify a dmenu-compatible launcher (default: dmenu)
 - $CM_OUTPUT_CLIP: if set, output clip selection to stdout
+- $CM_INPUT_CLIP: if set, accept clip selection from stdin
 EOF
     exit 0
-fi
+}
+
+list_clips() {
+    LC_ALL=C sort -rnk 1 < "$cache_file" | cut -d' ' -f2- | awk '!seen[$0]++'
+}
+
+print_sel(){
+    list_clips
+    exit 0
+}
+
+case "$1" in
+    # Not -h, see #142
+    --help) print_help ;;
+    --print-sel) print_sel ;;
```

Comment:
Hmm, why not implement `CM_LAUNCHER=cat` instead in a similar manner to fzf?

Reply:
Could do, But upon reviewing this code a bit more, I think I have a much better way of resolving this problem but it will need a brand new PR.
I'll quickly fix this, and when I have time I will post the new PR and see what you think.